### PR TITLE
chore: remove all unused currency-pair constants [BLO-1321]

### PR DESCRIPTION
**Pull request by g3ndo9kar9**
## In This PR
- I remove all `DefaultMarketConfigs` in each provider's `utils.go` folder. Previously, these assets were used for our `local` configs, but these no longer exist.
- I also remove the `currency_pairs.go` constant file as the majority of those values were unused + only used in test-files

### CODE REVIEW

File: oracle/orchestrator/helpers_test.go [hunk 1]
```diff
@@ -26,6 +26,11 @@ import (
 	mmtypes "github.com/skip-mev/slinky/x/marketmap/types"
 )
 
+var (
+	btcusdtcp = slinkytypes.NewCurrencyPair("BTC", "USDT")
+	ethusdtcp = slinkytypes.NewCurrencyPair("ETH", "USDT")
+)
```
Comment (aljo242) [inline oracle/orchestrator/helpers_test.go]:
```suggestion
var (
	btcusdtCP = slinkytypes.NewCurrencyPair("BTC", "USDT")
	ethusdtCP = slinkytypes.NewCurrencyPair("ETH", "USDT")
)
```

File: pkg/math/oracle/aggregator_test.go [hunk 2]
```diff
@@ -17,6 +16,12 @@ import (
 	mmtypes "github.com/skip-mev/slinky/x/marketmap/types"
 )
 
+var (
+	usdtusd = pkgtypes.NewCurrencyPair("USDT", "USD")
+	btcusd  = pkgtypes.NewCurrencyPair("BTC", "USD")
+	ethusd  = pkgtypes.NewCurrencyPair("ETH", "USD")
+)
```
Comment (aljo242) [inline pkg/math/oracle/aggregator_test.go]:
```suggestion
var (
	usdtusdCP = pkgtypes.NewCurrencyPair("USDT", "USD")
	btcusdCP = pkgtypes.NewCurrencyPair("BTC", "USD")
	ethusdCP = pkgtypes.NewCurrencyPair("ETH", "USD")
)
```

File: x/marketmap/types/market_test.go [hunk 3]
```diff
@@ -3,21 +3,18 @@ package types_test
...
-
-	"github.com/skip-mev/slinky/oracle/constants"
 	slinkytypes "github.com/skip-mev/slinky/pkg/types"
 	"github.com/skip-mev/slinky/providers/apis/coinbase"
 	"github.com/skip-mev/slinky/x/marketmap/types"
+	"github.com/stretchr/testify/require"
 )
 
 var (
+	btcusdtcp = slinkytypes.NewCurrencyPair("BTC", "USDT")
```
Comment (aljo242) [inline x/marketmap/types/market_test.go]:
```suggestion
	btcusdtCP = slinkytypes.NewCurrencyPair("BTC", "USDT")
```

File: tests/integration/slinky_suite.go [hunk 4]
```diff
@@ -542,9 +541,9 @@ func (s *SlinkyOracleIntegrationSuite) TestNodeFailures() {
 }
 
 func (s *SlinkyOracleIntegrationSuite) TestMultiplePriceFeeds() {
-	ethusdc := constants.ETHEREUM_USDC
-	ethusdt := constants.ETHEREUM_USDT
-	ethusd := constants.ETHEREUM_USD
+	ethusdc := slinkytypes.NewCurrencyPair("ETH", "USDC")
+	ethusdt := slinkytypes.NewCurrencyPair("ETH", "USDT")
+	ethusd := slinkytypes.NewCurrencyPair("ETH", "USD")
```
Comment (aljo242) [inline tests/integration/slinky_suite.go]:
```suggestion
	ethusdcCP := slinkytypes.NewCurrencyPair("ETH", "USDC")
	ethusdtCP := slinkytypes.NewCurrencyPair("ETH", "USDT")
	ethusdCP := slinkytypes.NewCurrencyPair("ETH", "USD")
```

File: tests/integration/slinky_suite.go [hunk 5]
```diff
@@ -344,7 +343,7 @@ func translateGRPCAddr(chain *cosmos.CosmosChain) string {
 }
 
 func (s *SlinkyOracleIntegrationSuite) TestNodeFailures() {
-	ethusdc := constants.ETHEREUM_USDC
+	ethusdc := slinkytypes.NewCurrencyPair("ETH", "USDC")
```
Comment (aljo242) [inline tests/integration/slinky_suite.go]:
```suggestion
	ethusdcCP := slinkytypes.NewCurrencyPair("ETH", "USDC")
```

**Review by aljo242 — APPROVED** [review]

**Review by davidterpay — APPROVED** [review]

**Comment by davidterpay** [comment]
I think the build should work now if u check argo

**Pull request merged**